src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -161,15 +161,16 @@ class UnifiedShareModal extends React.Component<Props, State> {
 
     constructor(props: Props) {
         super(props);
+        const inviteCollabsContacts = props.initiallySelectedContacts || [];
```

Comment:
Would it be better to add `initiallySelectedContacts` to defaultProps above and default it to []